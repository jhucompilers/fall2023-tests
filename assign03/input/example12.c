int main(void) {
  int x;
  x = 1;
  return -x;
}
