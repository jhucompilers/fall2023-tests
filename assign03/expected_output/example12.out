0|main|function|function () returning int
2|x|variable|int
